src/widgets/free-font/components/compare-floating-button/compare-floating-button.css.ts

```diff
@@ -0,0 +1,66 @@
+import { keyframes, style } from '@vanilla-extract/css';
+
+import { themeVars } from '@/shared/styles';
+const baseButtonStyle = style({
+  display: 'flex',
+  width: '100%',
+  height: '100%',
+  padding: '2.4rem 3.2rem',
```

Comment:
지금 피그마를 보면 이 플로팅 버튼은 width가 명확하게 지정되어 있는 것 같아요.
padding값들이 계속해서 바뀌는 것 보니 padding값을 기준으로 컴포넌트 UI를 만드신 게 아니라 width를 기준으로 만드신 것 같아요. 그래서 
```suggestion
  width: '18.8rem',
  padding: '2.4rem 0',
```

이렇게 수정해도 좋을 것 같아요 !